src/random_test.cpp

```diff
@@ -248,10 +248,15 @@ int sum_of_all_options(Thd1 *thd) {
     opt_int_set(ALTER_TABLESPACE_ENCRYPTION, 0);
     opt_int_set(ALTER_MASTER_KEY, 0);
     opt_int_set(ALTER_ENCRYPTION_KEY, 0);
+    opt_int_set(ALTER_GCACHE_MASTER_KEY, 0);
     opt_int_set(ROTATE_REDO_LOG_KEY, 0);
     opt_int_set(ALTER_DATABASE_ENCRYPTION, 0);
   }
 
+  /* Disable GCache encryption for MS or PS, only supported in PXC */
+   if (strcmp(FORK, "Percona-XtraDB-Cluster") != 0)
```

Comment:
better to disable it for PXC 2.4 also ?

Reply:
Good catch @rahulmalik87 .

Its already merged. I will push a post fix to address this. Please confirm if below fix looks good ?

  /* Disable GCache encryption for MS or PS, only supported in PXC-8.0 */
   if (strcmp(FORK, "Percona-XtraDB-Cluster") != 0 && server_version() < 80000)
      opt_int_set(ALTER_GCACHE_MASTER_KEY, 0);

Reply:
Good catch @rahulmalik87 . 

The PR is already merged, so I'll create a post fix and push the fix. Let me know if the below fix looks good ?
<img width="1062" alt="Screenshot 2022-09-21 at 11 41 26 AM" src="https://user-images.githubusercontent.com/50101455/191427547-ad6c55c2-d88a-40f5-8799-af0a52956d1d.png">